tasks/bug_report_template.rb

```diff
@@ -12,12 +12,23 @@
   end
 
   # Change Rails version if necessary.
-  gem "rails", "6.1.4"
-
-  gem "sprockets", "3.7.2"
-  gem "sassc-rails", "2.1.2"
-  gem "sqlite3", "1.4.2", platform: :mri
-  gem "activerecord-jdbcsqlite3-adapter", "61.0", platform: :jruby
+  gem "rails", "~> 7.0.0"
+
+  gem "sprockets", "~> 3.7"
+  gem "sassc-rails"
+  gem "sqlite3", platform: :mri
+  gem "activerecord-jdbcsqlite3-adapter", platform: :jruby
+
+  # Fixes an issue on CI with default gems when using inline bundle with default
+  # gems that are already activated
+  # Ref: rubygems/rubygems#6386
+  if ENV["CI"]
+    require "net/protocol"
+    require "timeout"
+
+    gem "net-protocol", Net::Protocol::VERSION
+    gem "timeout", Timeout::VERSION
+  end
```

Comment:
a little bit hacky, I've tried several things, like

- `bundle exec ruby`
- Move this check to a separate workflow and do not bundle before

It appears that at the moment that ruby is running, timeout gem with an older version is already activated, so the inline bundle check fails

Reply:
No issue with this at all. It didn't make much sense to me why it was happening. Just glad it's working now. Thanks again!